src/styles/index.scss

```diff
@@ -1,3 +1,76 @@
-body {
+@import './utils/variables';
+@import './utils/mixins';
+
+.page {
   margin: 0;
+  height: 100%;
+
+  @include block-centering;
+}
+
+.stopwatch {
+  height: $stopwatch-size;
+  width: $stopwatch-size;
+  border: 1vmin #000 dotted;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), hex color values should be written in lowercase. Please change `#000` to `#000`.